packages/typescript/index.bzl

```diff
@@ -124,7 +124,7 @@ def ts_project(
 
     Some TypeScript options affect which files are emitted, and Bazel wants to know these ahead-of-time.
     So several options from the tsconfig file must be mirrored as attributes to ts_project.
-    See https://www.typescriptlang.org/v2/en/tsconfig for a listing of the TypeScript options.
+    See https://www.typescriptlang.org/tsconfig for a listing of the TypeScript options.
```

Comment:
I thought i saw this updated here https://github.com/bazelbuild/rules_nodejs/commit/097732b513dba50f8c5351ecd0352d47f031a357

not sure what I did to loss the change.

Reply:
Oh sorry it just needed a rebase to go green then

Reply:
Do you have time to add a test of some sort? You might be able to just repro your issue in one of the existing tests

Reply:
I was able to ad some tests, but I will need help with providing the right `tsconfig.json` for the test case.